mypy/build.py

```diff
@@ -24,9 +24,17 @@
 import sys
 import time
 import types
-from collections.abc import Iterator, Mapping, Sequence, Set as AbstractSet
-from typing import TYPE_CHECKING, Any, Callable, ClassVar, Final, NoReturn, TextIO, TypedDict
-from typing_extensions import TypeAlias as _TypeAlias
+from collections.abc import Callable, Iterator, Mapping, Sequence, Set as AbstractSet
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    ClassVar,
+    Final,
+    NoReturn,
```

Comment:
Ideally should switch to Never in the long term, but can't yet because it was only added in 3.11